provider/idps/ldap.go

```diff
@@ -0,0 +1,123 @@
+package idps
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+)
+
+type LDAPIdentityProvider struct {
+	BindDN       types.String                    `tfsdk:"bind_dn"`
+	BindPassword types.String                    `tfsdk:"bind_password"`
+	CA           types.String                    `tfsdk:"ca"`
+	Insecure     types.Bool                      `tfsdk:"insecure"`
+	URL          types.String                    `tfsdk:"url"`
+	Attributes   *LDAPIdentityProviderAttributes `tfsdk:"attributes"`
+}
+
+type LDAPIdentityProviderAttributes struct {
+	EMail             []string `tfsdk:"email"`
```

Comment:
Can you update the type to be types.List?

Reply:
Done

Reply:
I think you also need to change the relevant code that set those values